src/global/constants.ts

```diff
@@ -18,3 +18,4 @@
  */
 
 export const EGO_JWT_KEY = 'EGO_JWT';
+export const LOGIN_NONCE = 'LOGIN_NONCE';
```

Comment:
This is part of the OIDC Auth Flow to prevent CSRF attacks https://developers.google.com/identity/openid-connect/openid-connect#sendauthrequest
The Cookie is auto-populated by the login flow so I've added logic to remove it on logout